### ISSUE TITLE
Alternative Download

### ISSUE THREAD

**Issue by IzzySoft** [frimtec/pikett-assist#158]
Would you mind making your app available outside the walled garden of Google Play – for those who cannot (or want-not) use Google? You could e.g. start attaching the APK to its corresponding tag/release. I'd then offer to serve it via [my repo](https://apt.izzysoft.de/fdroid) (updates are automatically fetched within 24h of your attaching the APK).

Thanks for considering!

**Comment by frimtec** [comment]
Hi there

I have added the APK to the latest release (https://github.com/frimtec/pikett-assist/releases/tag/1.4.8-Elephant).

Please be aware, PAssist requires an other app installed to work.
The SMS communication is done via S2MSP (https://github.com/frimtec/secure-sms-proxy). S2MSP  provides the APK packages with its GitHub releases as well. 
S2MSP is not available on Google Play as Google does not allow the permissions for SMS send/receive anymore on the Google Play since spring 2019.
S2MSP  is save and open source as well. It only requires SMS send/receive permissions, nothing else. It offers the SMS communication to other apps via an API on a phone number basis. Each phone number a app wants to send or receive SMS to/from must be approved by the user.

Please let me know if my apps are available on your repo. I will add a link to the PAssist documentation.

If you have further questions, just drop me a mail.

**Comment by IzzySoft** [comment]
Big thanks for all the details, @frimtec! And yes, both apps are fully eligible for my repo. I've just added them now (so they'll show up with the next sync tomorrow shortly before 8 pm CET). If you want to link to them, I even [have a badge](https://gitlab.com/IzzyOnDroid/repo#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it) you can use for that. The respective links would be:

* [S2MSP](https://apt.izzysoft.de/fdroid/index/apk/com.github.frimtec.android.securesmsproxy)
* [PAssist](https://apt.izzysoft.de/fdroid/index/apk/com.github.frimtec.android.pikettassist)

The description of PAssist directly links to S2MSP, to make it easier to find (on the website, it links to the website – inside the F-Droid app to the screen of the other app). PAssist will carry the NonFreeNet AntiFeature due to the Play BillingClient library, just in case you wonder – that library automatically pulls up the flag.

I guess having both available via my repo will make it easier for your clients to install them and keep them updated. As pointed out, updates will be pulled automatically (if you want to time it: the update checker runs at around 7:30 pm). Let me know if you'd like something on the description changed, or feel something else needed. If you want changes pointed out, the updater also grabs changelogs from Fastlane ([see here](https://gitlab.com/snippets/1895688) for my cheat-sheet on how to set up the essentials). Should you decide for that, please let me know when it's there so I can tell my updater to grab it.

Thanks a lot for your fast response!

**Comment by frimtec** [comment]
@IzzySoft I have updated PAssist on Wednesday at around 18:00 to 1.4.9-Elephant. But it was not picked up yet. Is their something I need to take care of?
Is it maybe the name in my version schema?

**Comment by IzzySoft** [comment]
@frimtec thanks for alerting me – and apologies, that was my fault! Minor misspelling in the "tag schema" resulted in my updater ignoring all tags. Fixed and verified, updater now correctly recognizes your tags and just pulled the update. For compensation (and so you don't have to wait another day), I just manually triggered an index build and sync, so the update should appear in a few minutes (5..10, raw estimate).

On the next update it should work as expected – and my mouse should pick your Elephant :smiley:
